docs/developer-guide/rules.md

```diff
@@ -189,7 +189,10 @@ Each rule must be accompanied by a README, fitting the following format:
 7.  Example patterns that are *not* considered violations (for each option value).
 8.  Optional options (if applicable).
 
-Look at the READMEs of other rules to glean more conventional patterns.
+Look at the READMEs of other rules to glean more conventional patterns. These includes:
+
+-   Aligning the arrows within the prototypical code example with the beginning of the construct being highlighted.
```

Comment:
I think this would benefit from an example. E.g.
````diff
 @media screen and (min-width: 768px) {}
-/**                    ↑        ↑
+/**                ↑          ↑
  *         These features and these values */
````

Reply:
Yep. This also highlights another subtle convention... the description should be as left aligned as possible e.g.

```css
 @media screen and (min-width: 768px) {}
/**                 ↑          ↑
  *       These names and values */
```

I'll add an example and additional bullet.